package.json

```diff
@@ -50,7 +50,8 @@
     "npm": ">=6"
   },
   "dependencies": {
-    "@tds/core-responsive": "^1.1.0"
+    "@tds/core-responsive": "^1.1.0",
```

Comment:
I cannot find `@tds/core-responsive` being used in _PreviewCard.jsx_ below. Is this dependency required?

Reply:
This dependency is the root package.json file, being used by other packages.  It says I committed it because of the added comma since I added styled-components underneath.